pkg/virt-handler/vm.go

```diff
@@ -258,6 +261,27 @@ func NewController(
 		return nil, err
 	}
 
+	_, err = vmiSourceInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Comment:
What about the `AddFunc`?

Reply:
Since we need to launch the server when the VMI is running, getting an Update notification is enough